### PR TITLE
Docs: Restructure configure thresholds docs

### PR DESCRIPTION
**Pull request by imatwawana**
This PR does the following:

- Removes redundant introduction and About thresholds heading
- Adds Threshold options heading with Threshold mode and Show thresholds sub-headings
- Adds placeholder text in unpopulated sections
- Changes heading Add or delete a threshold to Add a threshold, moves delete content after task, and reformats task

**Out of scope**: Content additions or copy edits; those will be done on a new PR branched off this branch; then merged back in.

**TO DO**

- [x]  Get this PR approved
- [x]  Get PR #81520 approved (content additions & updates)
- [x]  Merge PR #81520 into this branch

Then merge this PR

### CODE REVIEW

File: docs/sources/panels-visualizations/configure-thresholds/index.md [hunk 1]
```diff
@@ -53,24 +44,39 @@ On visualizations that support it, Grafana sets default threshold values of:
 
 The **Base** value represents minus infinity. It is generally the “good” color.
 
-## Add or delete a threshold
+## Threshold options
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/configure-thresholds/index.md:47]:
```suggestion
## Thresholds options
```

File: docs/sources/panels-visualizations/configure-thresholds/index.md [hunk 2]
```diff
@@ -53,24 +44,39 @@ On visualizations that support it, Grafana sets default threshold values of:
...
 
-You can add as many thresholds to a panel as you want. Grafana automatically sorts thresholds values from highest to lowest.
+TBA
 
-Delete a threshold when it is no longer needed. When you delete a threshold, the system removes the threshold from all visualizations that include the threshold.
+### Threshold mode
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/configure-thresholds/index.md:51]:
```suggestion
### Thresholds mode
```

File: docs/sources/panels-visualizations/configure-thresholds/index.md [hunk 3]
```diff
@@ -53,24 +44,39 @@ On visualizations that support it, Grafana sets default threshold values of:
...
 
-1. To add a threshold:
+There are two types of thresholds:
 
-   a. Edit the panel to which you want to add a threshold.
+- **Absolute** thresholds are defined by a number. For example, 80 on a scale of 1 to 150.
+- **Percentage** thresholds are defined relative to minimum or maximum. For example, 80 percent.
+
+### Show threshold
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/configure-thresholds/index.md:58]:
```suggestion
### Show thresholds
```

**Review by nmarrs — APPROVED** [review]
✏️✅

**Review by codeincarnate — APPROVED** [review]
Looks good!

File: docs/sources/panels-visualizations/configure-thresholds/index.md [hunk 4]
```diff
@@ -53,24 +44,39 @@ On visualizations that support it, Grafana sets default threshold values of:
 
 The **Base** value represents minus infinity. It is generally the “good” color.
 
-## Add or delete a threshold
+## Thresholds options
 
-You can add as many thresholds to a panel as you want. Grafana automatically sorts thresholds values from highest to lowest.
+TBA
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/panels-visualizations/configure-thresholds/index.md:49]: Neither actually. PR #81520 is branched off this branch and that's where all the content edits are happening including filling in those TBA sections. I do separate PRs for the restructuring and the content additions to make them faster and easier to review.

[the remark this answers is not in the document]
Comment (baldm0mma) [reply docs/sources/panels-visualizations/configure-thresholds/index.md:49]: lovely!

**Review by baldm0mma — APPROVED** [review]

**Review by baldm0mma — APPROVED** [review]

**Review by baldm0mma — APPROVED** [review]
Once this Drone error is fixed, everything looks great!

**Pull request merged**